nixos/modules/tasks/filesystems/zfs.nix

```diff
@@ -75,6 +79,23 @@ in
           '';
       };
 
+      enableCryptoStability = mkOption {
+        type = types.bool;
+        default = false;
```

Comment:
Options should be inverted here and provide an option to use the old version instead. We want new users to receive the new version by default - otherwise they have to migrate later.
Therefore, `enableCryptoStability` should be named `enableLegacyCryptoFormat`.

Reply:
I did make those changes now. I also altered description for the config options accordingly.